ios/RNSScreen.m

```diff
@@ -333,7 +458,9 @@ - (void)willMoveToParentViewController:(UIViewController *)parent
 - (void)viewWillAppear:(BOOL)animated
 {
   [super viewWillAppear:animated];
-
+  [UIView animateWithDuration:0.5 animations:^{
```

Comment:
why 0.5? If this is needed then please add some comment to explain this

Reply:
It is an arbitrary value taken from some SO thread. `Programming iOS 13` book uses a 0.4-second long transition but also doesn't specify why such value exactly, so I have no answer for this.

Reply:
we should have that note included in the code as a comment then (preferably including the link to SO answer)